ledger/src/blockstore_processor.rs

```diff
@@ -154,6 +154,20 @@ fn aggregate_total_execution_units(execute_timings: &ExecuteTimings) -> u64 {
     execute_cost_units
 }
 
+fn sum_additional_costs(batch: &TransactionBatch) -> u64 {
```

Comment:
I would prefer we didn't sum up all these miscellaneous costs for v1. I don't want all these other costs getting grouped together under the same units as execution cost

Reply:
If we wanted, for instance, in v2 to limit the number of sequential operations, then we could have a maximum compute per account tracked in a separate data structure. But I don't think they should be mixed into the same unit of measurement.

Reply:
I think here Jason sums up the CUs accountable for those "const" costs - sigverify, write lock, etc. These CUs are needed if to check against `block_limit`. `Account_limit` is not yet checked in this version.

Jason early brought up a good concern that if it's possible a leader packed a block to `block_limit + 1` CUs (given all the estimated/actual cost adjustment, it is possible), should replay_stage therefore abandon this block because it is 1CU above limit? Probably not. Maybe we want to have replay to use a different limit. If that is the case, then we may as well set up a `bpf execution limit` for replay, then we don't need to sum up the other costs. Just this `bpf execution limit` would be total arb.

Reply:
Carl, I think these are already all in the same unit of measurement. The original code totaled the cu used by the transaction's code, but did not include the static costs, which ARE totaled by the leader when creating the block.